submissions/maxim-zabolotny/oop-exersice/index.js

```diff
@@ -0,0 +1,79 @@
+class Creature {
+    constructor(name, gender, species, legs, saying) {
+      this.name = name;
+      this.gender = gender;
+      this.species = species;
+      this.legs = legs;
+      this.saying = saying;
+      this.friends = [];
+    }
+  
+    addFriends(...newFriends) {
+      newFriends.forEach(({ name }) => this.friends.push(name));
+      return this.friends;
+    }
+  
+    toString() {
+      let template = [
+        `name: ${this.name}; `,
+        `gender: ${this.gender}; `,
+        `species: ${this.species}; `,
+        `saying: ${this.saying}; `,
+        `legs: ${this.legs}; `,
+      ];
```

Comment:
Does it really contain template?
Each fragment here ends with `; `
Why not to use this repeating delimiter in `join` below?